app/models/container_node.rb

```diff
@@ -72,17 +73,35 @@ def perf_rollup_parents(interval_name = nil)
     [ext_management_system] unless interval_name == 'realtime'
   end
 
-  def ipaddress
+  def kubernetes_hostname
     labels.find_by(:name => "kubernetes.io/hostname").try(:value)
   end
 
   def cockpit_url
-    URI::HTTP.build(:host => ipaddress, :port => 9090)
+    URI::HTTP.build(:host => kubernetes_hostname, :port => 9090)
   end
 
   def evaluate_alert(_alert_id, _event)
     # currently only EmsEvents from hawkular are tested for node alerts,
     # and these should automaticaly be translated to alerts.
     true
   end
+
+  supports :external_logging_support do
+    unless ext_management_system.respond_to?(:external_logging_route_name)
+      unsupported_reason_add(:external_logging_support, _('This provider type does not support Extrenal Logging'))
+    end
+  end
+
+  def external_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def external_logging_path
+    fqdn = (kubernetes_hostname || name || "")
+    hostname = fqdn.split('.').first || ""
+    query = "bool:(filter:(or:!((term:(hostname:'#{hostname}')),(term:(hostname:'#{fqdn}')))))"
```

Comment:
@simon3z, unfortunately, the logging subsystem does not return a consistent set of hostnames for the same entity.  So some parts of the logging subsystem gather the FQDN, other parts might use the shortened name.  I believe this query is trying to account for that.  We have to file bugs with the logging subsystem to fix the collectors to ensure an FQDN.

Having said that, data collection is never going to be perfect, so it is likely we'll need to keep this around.